examples/bull-spread/bull-spread.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"cloud.google.com/go/civil"
+	"github.com/shopspring/decimal"
+
+	"github.com/alpacahq/alpaca-trade-api-go/v3/alpaca"
+	"github.com/alpacahq/alpaca-trade-api-go/v3/marketdata"
+)
+
+var decimal10k = decimal.NewFromInt(10000)
+
+const (
+	oneWeek = 7
+)
+
+type Service struct {
+	tdClient *alpaca.Client
+	mdClient *marketdata.Client
+	acct     *alpaca.Account
+}
+
+func NewService() (*Service, error) {
```

Comment:
[minor] I don't see the value of this struct. It's possible that you wanted to follow the previous examples in this repo, but honestly they are not to be followed.

Reply:
> It's possible that you wanted to follow the previous examples

No, I checked a few, they use init and such, so it wasn't my intention to get inspiration from those,
I merely wanted to encapsulate the services used by example and allow for easy extension.

Reply:
I'd prefer not having it, but at the very least could you please give it a better name?

Reply:
Fixed
a0545df8bfa0e00b8922f8b06424d92b6fc7995a